bot_management/bot_body.py

```diff
@@ -0,0 +1,126 @@
+from time import sleep, ctime
+import helpfull_functions
+from binance.client import Client
+from binance.enums import *
+from binance.exceptions import BinanceAPIException, BinanceOrderException
+from downloader.global_variables import *
+
+
+class Bot():
+
+    def __init__(self, bot_id):
+        self.running = True
+        self.bot_id = bot_id
+
+    def switcher(self, bot_id, binance_client, amount, what_to_do="waiting", test="0"):
+        """bot function that does all what bot does"""
+        client = helpfull_functions.setup_client("localhost", "8086", "test_bot_logs")
+        binance_client.API_URL = 'https://testnet.binance.vision/api'
+        json_body = [
+            {
+                "measurement": "bot_action",
+                "tags": {
+                    "id": bot_id
+                },
+                "time": ctime(),
+                "fields": {
+                    "action": what_to_do,
+                    "amount": amount
+                }
+            }]
+
+        if what_to_do == "invest":
+            if test == 2:
+                # only needed for tests
+                array_with_my_data.append(bot_id)
+            elif test == 3:
+                try:
+                    buy_order = binance_client.create_test_order(
+                        symbol='BTCUSDT',
+                        side=SIDE_BUY,
+                        type=ORDER_TYPE_MARKET,
+                        quantity=amount)
+                    client.write_points(json_body)
+                except BinanceAPIException as e:
+                    # error handling goes here
+                    return e
+                except BinanceOrderException as e:
+                    # error handling goes here
+                    return e
+            else:
+                try:
+                    buy_order = binance_client.create_order(
+                        symbol='BTCUSDT',
```

Comment:
I can see a hardcoded symbol here. Why do we have such things hardcoded? User should have elasticity to define what he/she wants.